src/Compiler/RefC/RefC.idr

```diff
@@ -1024,7 +1026,7 @@ footer = do
 export
 executeExpr : Ref Ctxt Defs -> (execDir : String) -> ClosedTerm -> Core ()
 executeExpr c _ tm
-    = do coreLift_ $ putStrLn "Execute expression not yet implemented for refc"
+    = do log "compiler.refc" 0 "Execute expression not yet implemented for refc"
```

Comment:
I think this one is a valid `putStrLn`: it's an error message displayed upon failure.

Reply:
I'll take your word for it. Updated